examples/package.json

```diff
@@ -5,22 +5,25 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "build": "yarn build:clean-state && yarn build:counter && yarn build:counter-lowlevel && yarn build:cross-contract-call && yarn build:fungible-token-lockable && yarn build:fungible-token && yarn build:non-fungible-token && yarn build:status-message-collections && yarn build:status-message",
+    "build": "yarn build:clean-state && yarn build:counter && yarn build:counter-lowlevel && yarn build:counter-ts && yarn build:cross-contract-call && yarn build:fungible-token-lockable && yarn build:fungible-token && yarn build:non-fungible-token && yarn build:status-message-collections && yarn build:status-message",
     "rebuild": "cd .. && yarn build && cd examples && rm -rf node_modules && rm -rf build && yarn && yarn build",
     "build:status-message": "yarn near-sdk build src/status-message.js build/status-message.wasm",
     "build:clean-state": "near-sdk build src/clean-state.js build/clean-state.wasm",
     "build:counter": "near-sdk build src/counter.js build/counter.wasm",
     "build:counter-lowlevel": "near-sdk build src/counter-lowlevel.js build/counter-lowlevel.wasm",
+    "build:counter-ts": "near-sdk build src/counter.ts build/counter-ts.wasm",
     "build:cross-contract-call": "near-sdk build src/status-message.js build/status-message.wasm && near-sdk build src/cross-contract-call.js build/cross-contract-call.wasm",
     "build:fungible-token-lockable": "near-sdk build src/fungible-token-lockable.js build/fungible-token-lockable.wasm",
     "build:fungible-token": "near-sdk build src/fungible-token.js build/fungible-token.wasm",
     "build:non-fungible-token": "near-sdk build src/non-fungible-token-receiver.js build/non-fungible-token-receiver.wasm && near-sdk build src/non-fungible-token.js build/non-fungible-token.wasm",
     "build:status-message-collections": "near-sdk build src/status-message-collections.js build/status-message-collections.wasm",
-    "test": "ava && yarn test:counter-lowlevel",
+    "test": "ava && yarn test:counter-lowlevel && yarn test:counter-ts && yarn test:ts",
+    "test:ts": "yarn run tsc",
```

Comment:
I guess this command will not execute ts tests.

Reply:
And I do not think we need a separate test command for TS-based scripts.

Reply:
Oh my naming is wrong. it's to type checking the code with tsc without building or testing. The result, is same as what is given by VSCode shows inline.

We need this because babel typescript plugin doesn't check errors. Maybe it's better to do it by `tsc --noEmit` as first step of `near-sdk-js build`, when typescript contract is given?

Reply:
> Maybe it's better to do it by tsc --noEmit as first step of near-sdk-js build, when typescript contract is given?

I did the simple fix for now. If you think ^ is better will do in next PR.